api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.7.0"
```

Comment:
The file at HEAD of the main repo https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json is saying 1.7.0, the file i generated says 1.6.0. I am not sure how to go forward with this one.

Once i finalize all the code comments, I will re-gen everything again and commit it as a separate commit.